myproject/myproject/settings.py

```diff
@@ -31,17 +31,25 @@
 # Application definition
 
 INSTALLED_APPS = [
+    # Default Django apps
     'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+
+    # User authentication app
     'accounts',
+
+    # Django REST Framework for building APIs
     'rest_framework',
+
+    # Token authentication support for DRF
     'rest_framework.authtoken',
```

Comment:
instead of adding comments you can do as 
django_apps = [...]
local_apps = [...]
others_apps = [...]

INSTALLED_APPS = django_apps  + local_apps + others_apps